### DISCUSSION TITLE
Best way to run macOS tests with Apple Silicon?

### DISCUSSION THREAD

**Discussion by erikkerber** [bazelbuild/rules_apple#1513] [Q&A] [answered]
I'm looking to run tests locally on Apple Silicon, similar to the "Latest Bazel on 🍎" steps.

Using Bazel 5.2.0:
```
 bazel build --test_tag_filters=-apple_static_xcframework,-apple_xcframework_import  --  test/...
```

Fails because the tests expect x86_64:

```
DEBUG: /private/var/tmp/_bazel_ekerber/572a16df961f98c6126d78b8f1b8e177/external/bazel_skylib/lib/unittest.bzl:361:10: In test _linkmap_test_impl from //test/starlark_tests:rules/linkmap_test.bzl: Expected

test/starlark_tests/targets_under_test/ios/app_x86_64.linkmap

to be built. Contents were:

test/starlark_tests/targets_under_test/ios/app.ipa
test/starlark_tests/targets_under_test/ios/app_arm64.linkmap
```

I tried passing `--ios_multi_cpus` and `--macos_multi_cpus`, but the build then fails on `test/starlark_tests/macos_kernel_extension`

Any other flags or workarounds outside of sending these tests to CI?

**Comment by keith** [comment]
Disabling the kext thing here https://github.com/bazelbuild/rules_apple/pull/1515

**Comment by keith** [comment, marked as answer]
Here's a fix for the arch issue https://github.com/bazelbuild/rules_apple/pull/1516
